parastell/cubit_utils.py

```diff
@@ -63,8 +64,6 @@ def export_step_cubit(filename, export_dir=""):
         export_dir (str): directory to which to export the STEP output file
             (defaults to empty string).
     """
-    init_cubit()
```

Comment:
I think we included these to be safe.  By removing them, I guess we require folks to be sure they have called `init_cubit()` or they'll get an error?

Reply:
Yes, which I think makes sense for the functions that require certain actions within Cubit to have been taken (e.g., exports, imprinting, merging, etc.).